source/version.js

```diff
@@ -55,3 +55,24 @@ module.exports.validate = version => {
 		throw new Error('Version should be a valid semver version.');
 	}
 };
+
+module.exports.satisfiesRequiredVersion = async function (pkgName, installedVersion) {
+	const range = requirements[pkgName];
+	if (!satisfies(installedVersion, range)) {
+		throw new Error(`Please upgrade ${pkgName} to ${range} or higher.`);
+	}
+};
+
+/**
+ * Check whether the given version satisfies the range
```

Comment:
There is no need to document the method using JSDoc as we're not using it (the API specification) anyways.